tests/test_backend.py

```diff
@@ -67,3 +67,41 @@ class TestBackend(PyRDF.backend.Backend.Backend):
 
         with self.assertRaises(TypeError):
             TestBackend()
+
+class OperationSupportTest(unittest.TestCase):
+    """
+    Test cases to ensure that incoming operations are
+    classified accurately in local environment.
+
+    """
```

Comment:
We need to test all the operations that are not supported by every backend.
In the case of local, we need to test that Take, Snapshot and Foreach raise an exception. Also, we need to test that Range only raises an exception with the local backend in multi-threaded mode, but not in sequential mode.

Reply:
Ok @etejedor . Will add them :+1: